pegasus/sites.v3/hourofcode.com/public/index.haml

```diff
@@ -87,8 +91,14 @@ social:
                 = view :index_video
                 %div{style: "clear: both;"}
               #leftbutton.visible-xs
-                %h1.front-header-banner.watch-video=hoc_s(:front_watch_regular_video)
-                %img{src: "/images/play-button.png", width: "60", onclick: "return startVideo();", style: "cursor: pointer;"}
+                -# If language Spanish or Brazilian Portuguese, show alternate
+                -# promo video for LATM Hour of Code.
+                - if latam_language_codes.include?(@language) && show_latam_videos
+                  %h1.front-header-banner.watch-video=hoc_s(:front_watch_regular_video)
+                  %img{src: "/images/play-button.png", width: "60", onclick: "return startLATMVideo();", style: "cursor: pointer;"}
```

Comment:
Typo `startLATMVideo`

Reply:
Ah! Good catch - I did that with the branch names too 🙃